modules/aws-eks-addon/aws-ebs-csi-driver/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "cluster_id" {
+  type = string
+}
+
+variable "common_tags" {
+  type = map(string)
+}
+
+variable "eks_addon_aws_ebs_csi_driver_config" {
```

Comment:
How do you feel about dropping the `eks_addon_aws_ebs_csi_driver` prefix? Can just call this `add_on_config`.

See here for other notes on var names https://github.com/aws-ia/terraform-ssp-amazon-eks/projects/1

Reply:
This config is unique to each EKS Add-on similar to Kuberentes-addons. We cannot have the same variable( `add_on_config` ) for multiple add-ons. Also, map of maps is not a clean solution for EKS Add-ons

eg:

```
 enable_eks_addon_kube_proxy = true # default is false
  #Optional
  eks_addon_kube_proxy_config = {
    addon_name               = "kube-proxy"
    addon_version            = "v1.21.2-eksbuild.2"
    service_account          = "kube-proxy"
    resolve_conflicts        = "OVERWRITE"
    namespace                = "kube-system"
    additional_iam_policies  = []
    service_account_role_arn = ""
    tags                     = {}
  }

  enable_eks_addon_aws_ebs_csi_driver = true # default is false
  #Optional
  eks_addon_aws_ebs_csi_driver_config = {
    addon_name               = "aws-ebs-csi-driver"
    addon_version            = "v1.4.0-eksbuild.preview"
    service_account          = "ebs-csi-controller-sa"
    resolve_conflicts        = "OVERWRITE"
    namespace                = "kube-system"
    additional_iam_policies  = []
    service_account_role_arn = ""
    tags                     = {}
  }
```